apps/baseai.dev/src/components/Header.tsx

```diff
@@ -67,21 +71,53 @@ function HeaderLinks() {
 
 	return (
 		<>
-			<Anchor
-				href={'https://github.com/LangbaseInc/baseai'}
-				target="_blank"
-				className="hidden w-auto font-semibold lg:flex "
-				variant={'ghost'}
-			>
-				★ BaseAI
-			</Anchor>
 			<Anchor
 				href={href}
 				className="w-28 font-semibold"
 				variant={'default'}
 			>
 				{text}
 			</Anchor>
+			<Anchor
```

Comment:
We have variants in Anchor component. Can you please use one of them instead of defining custom styles? I think it will be outline variant.

Reply:
We need a thicker stroke then defined in the variant. We also need custom styles for background.